.github/workflows/soaking.yml

```diff
@@ -14,7 +14,7 @@
 name: 'weekly soaking test'
 on:
   schedule:
-    - cron: '0 10 * * SAT' # every Saturday at 10 am UTC: pst 2am
+    - cron: '0 10 * * SAT' # every Saturday at 10 am UTC: pst 2am'0 10 * * SAT'
```

Comment:
Why add the cron in the comment?

Reply:
Another typo from me. Should be deleted by now.